wavelength-plugins/tsumeta.js

```diff
@@ -0,0 +1,180 @@
+/*****************************
+ * TsuMeta Committee Plug-in *
+ * Created by Insist         *
+ * Idea/Made for Desokoro    *
+ *****************************/
+
+"use strict";
+
+let proposals = {};
+
+const fs = require('fs');
```

Comment:
I want to start using lib/fs.js for all new plugins, and refactor the old ones later

Reply:
Oh, okay.